services/track_visit_service.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../models/page'
+
+class TrackVisitService
+  def initialize(page)
+    @page = page
+  end
+
+  def track
+    PageVisit.create(visited_at: DateTime.now, page_id: @page.id)
```

Comment:
```suggestion
    PageVisit.create(visited_at: Time.now.utc, page_id: @page.id)
```

It is good practice to save the date in UTC in the DB